design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -0,0 +1,261 @@
+}
+
+type DownloadableRequisitionListItem implements RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
+    product: ProductInterface!
+    "Quantity added"
+    qty: Float!
+    "custom Option selected"
+    customizable_options: [SelectedCustomizableOption]
+    "DownloadableProductLinks defines characteristics of a downloadable product"
+    links: [DownloadableProductLinks]
+    "DownloadableProductSamples defines characteristics of a downloadable product"
+    samples: [DownloadableProductSamples]
+}
+
+type BundleRequisitionListItem implements RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
+    product: ProductInterface!
+    "Quantity added"
+    qty: Float!
+    "custom Option selected"
+    customizable_options: [SelectedCustomizableOption]
+    "selected bundle options"
+    bundle_options: [SelectedBundleOption]!
+}
+
+type ConfigurableRequisitionListItem implements RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
+    product: ProductInterface!
+    "Quantity added"
+    qty: Float!
+    "custom Option selected"
+    customizable_options: [SelectedCustomizableOption]
+    "Configurable options selected"
+    configurable_options: [SelectedConfigurableOption]
+}
+
+type Mutation {
+    "Create Empty Requisition List"
+    createRequisitionList(
+        "name for the list"
+        name: String!,
+        "description For the list"
+        description: String
+    ): RequisitionList
+
+    "Rename a requisition list"
+    renameRequisitionList(
+        "unique Id of requisition list"
+        id: ID!,
+        "new name for list"
+        name: String!,
+        "new description For the List"
+        description: String!
+    ): RequisitionList
+
+    DeleteRequisitionList(
+        "unique Id of requisition list"
+        id: ID!
+    ): RequisitionList
+
+    "Add items to requisition list"
+    addProductsToRequisitionList(
+        "unique Id of requisition list"
+        id: ID!,
+        "Products to be added to requisition list"
+        items: [RequisitionListItemsInput!]!
+    ): AddProductsToRequisitionListOutput
+
+    "Remove Items in requisition list"
+    removeRequisitionListItems(
+        "unique Id of requisition list"
+        id: ID!,
+        "unique Ids of Items to be removed from requisition list"
+        items: [ID!]!
+    ): RemoveRequisitionListItemsOutput
+
+    "Update Items in requisition list"
+    updateRequisitionListItems(
+        "unique Id of requisition list"
+        id: ID!,
+        "Items to be updated from requisition list"
+        items: [UpdateRequisitionListItemsInput!]!
+    ): UpdateRequisitionListItemsOutput
+
+    "Add Requisition List Items To Cart"
+    addRequisitionListItemToCart(
+        "Cart Id to which requisition list needs to copied"
+        cart_id: String!,
```

Comment:
If requisition lists are only supported by registered customers, `cart_id` can be calculated based on the auth token.